### DISCUSSION TITLE
TAPO C210 HD stream 3MP/1080p

### DISCUSSION THREAD

**Discussion by jsapede** [JurajNyiri/HomeAssistant-Tapo-Control#718] [Q&A] [answered]
hi,
in the tapo app one can choose if hd streeam is 3MP or 1080p. is there a way to switch this with th integration ?

**Comment by JurajNyiri** [comment, marked as answer]
No, integration uses rtsp streams exposed by camera. There are 2, one HQ and one LQ.
